WebAPI.OData.EF6/Startup.cs

```diff
@@ -72,6 +73,14 @@ private IEdmModel GetEdmModel()
             var builder = new ODataConventionModelBuilder();
             //builder.Namespace = "com.FooBar";
             builder.EntitySet<OpsTenant>(nameof(OpsTenant));
+            builder.EntityType<OpsTenant>()
```

Comment:
I think the changes here should be as follows so that the current web tests don't have to change.
```c#
            builder.EntitySet<OpsTenant>(nameof(OpsTenant));
            builder.EntityType<OpsTenant>()
                .Collection
                .Function("AllowApplyAttribute")
                .ReturnsCollectionFromEntitySet<OpsTenant>(nameof(OpsTenant));

            builder.EntitySet<CoreBuilding>(nameof(CoreBuilding));
            builder.EntityType<CoreBuilding>()
                .Collection
                .Function("AllowApplyAttribute")
                .ReturnsCollectionFromEntitySet<CoreBuilding>(nameof(CoreBuilding));
```

Reply:
I'll look into this

Reply:
Updated as requested